packages/esm-patient-labs-app/src/lab-orders/add-lab-order/test-type-search.component.tsx

```diff
@@ -45,6 +46,13 @@ export function TestTypeSearch({ openLabForm }: TestTypeSearchProps) {
     searchInputRef.current?.focus();
   };
 
+  const cancelOrder = useCallback(() => {
+    closeWorkspace('add-lab-order', {
+      ignoreChanges: true,
```

Comment:
I'm not sure if we need to ignore changes here."

https://github.com/user-attachments/assets/a47d23c5-ea89-4729-b60a-54fed0ea0d54

Reply:
It might be more convenient to return to the search results when on the form, rather than going back to the order basket. Just a thought.

```typescript
const cancelOrder = useCallback(() => {
    if (currentLabOrder) {
      setCurrentLabOrder(null);
    } else {
      closeWorkspace({
        ignoreChanges: true,
        onWorkspaceClose: () => launchPatientWorkspace('order-basket'),
      });
    }
  }, [closeWorkspace, currentLabOrder]);
```

Reply:
Not sure I understand. Mind suggesting a change? @usamaidrsk?

Reply:
I will suggest this later as a follow up on this PR. Overall suggestion is to return to the search results if one is on the drug or lab forms respectively.